CHANGELOG

```diff
@@ -1,4 +1,14 @@
 # Changelog
+## [0.2.4] - 2020-02-12
+### Bug Fixes
+ - Reduce minimal required MAC address length to 4 octets
+ - Raise NmstateTimeoutError on mainloop timeout
+ - Validate duplicate interface names
+ - Support ovs-bridge and ovs-interface with the same name
```

Comment:
This is not supported, we only have a test that shows that it is not supported. This line can be dropped IMHO.

Reply:
Done, thanks!